test/functional/JIT_Test/src/jit/test/jitt/codecache/TestManager.java

```diff
@@ -282,8 +282,8 @@ private static boolean isAOTVMOptionsSet() {
 	 * @return true if -Xshareclasscache VM option is in use, false otherwise
 	 */
 	public static boolean isShareClassesSet() {
-		RuntimeMXBean RuntimemxBean = ManagementFactory.getRuntimeMXBean();
-		List<String> aList=RuntimemxBean.getInputArguments();
+		RuntimeMXBean RuntimemxBean = (RuntimeMXBean)ManagementFactory.getRuntimeMXBean();
+		List<String> aList=RuntimemxBean.getAllInputArguments();
```

Comment:
How exactly should I implement a helper method here? In my opinion, this might be out of the scope of the PR

Reply:
Rather than a helper method, I was thinking of declaring `RuntimemxBean` and `aList` as a variable/member of the class. I would call getAllInputArguments() only once.